tests/test_modify_swagger_auth.py

```diff
@@ -8,27 +8,30 @@
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from scripts.swagger_auth import SecureSwagger, determine_auth_configuration_from_swagger
+from scripts.swagger_auth import SecureSwagger
 
 
 class TestSecureSwagger(unittest.TestCase):
     def setUp(self):
         self.secure_auth = os.path.join(pkg_root, 'auth.secure_all.json')
-        self.hca_default_auth = os.path.join(pkg_root, 'auth.hca_default.json')
+        self.hca_default_auth = os.path.join(pkg_root, 'auth.default.json')
+        self.swagger_path = os.path.join(pkg_root, 'dss-api.yml')
 
         # store the current swagger contents since we'll be mutating them in the tests
-        with open(os.path.join(pkg_root, 'dss-api.yml'), 'r') as f:
+        with open(self.swagger_path, 'r') as f:
             self.original_contents = f.readlines()
 
     def tearDown(self):
         # restore the original swagger contents
-        with open(os.path.join(pkg_root, 'dss-api.yml'), 'w') as f:
+        with open(self.swagger_path, 'w') as f:
             for line in self.original_contents:
                 f.write(line)
 
     def test_auth_can_be_determined_from_swagger(self):
```

Comment:
where do you test for an empty security config file, a security config file with non-existant paths, and a security config with duplicates?